examples_gbs/run_tutorial_sample.py

```diff
@@ -0,0 +1,140 @@
+# pylint: disable=invalid-name,no-member,wrong-import-position,wrong-import-order,ungrouped-imports
+"""
+Sampling Tutorial
+=================
+
+This tutorial provides a quick guide to generating samples from GBS. This functionality is
+provided in the :mod:`~.gbs.sample` module of the applications layer.
+
+Sampling from GBS
+-----------------
+
+A GBS device can be programmed to sample from any symmetric matrix :math:`A`. To sample,
+we must specify both the mean number of photons being generated in the device and the form of
+detection used at the output: threshold detection or photon-number resolving (PNR) detection.
+Threshold detectors are restricted to measuring whether or not one or more photons have arrived
+at the detector, whereas PNR detectors are able to count the number of photons.
+
+Let's take a look at both types of sample. We can generate samples from a random 5-dimensional
+matrix:
+"""
+
+from strawberryfields.gbs import sample
+import numpy as np
+
+modes = 5
+n_mean = 4
+samples = 5
+
+A = np.random.random((modes, modes))

```

Comment:
```suggestion
A = np.random.normal(0, 1, (modes, modes))
```
I think this is a good opportunity to include matrices with negative entries and negative eigenvalues